gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy

```diff
@@ -21,16 +21,35 @@ import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
 import com.palantir.gradle.dist.service.JavaServiceDistributionPlugin
 import groovy.transform.EqualsAndHashCode
 import groovy.transform.ToString
+import java.nio.file.Files
 import org.gradle.api.DefaultTask
 import org.gradle.api.file.FileCollection
-import org.gradle.api.tasks.*
-
-import java.nio.file.Files
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
 
 class LaunchConfigTask extends DefaultTask {
 
     static final List<String> tmpdirJvmOpts = ['-Djava.io.tmpdir=var/data/tmp']
 
+    static final Map<String, List<String>> gcProfiles = [
+            'throughput'   : ['-XX:+UseParallelOldGC'],
+            'response-time': ['-XX:+UseParNewGC',
+                              '-XX:+UseConcMarkSweepGC',
+                              '-XX:+CMSParallelInitialMarkEnabled',
+                              '-XX:+CMSParallelRemarkEnabled',
```

Comment:
We probably don't need to set `-XX:+CMSParallelInitialMarkEnabled -XX:+CMSParallelRemarkEnabled` as those should default to true with Java 8+, see https://github.com/AdoptOpenJDK/openjdk-jdk8u/blame/fcb5661ae078d0e3521a1237a67b3039b1bfdbe3/hotspot/src/share/vm/runtime/globals.hpp#L1792